source/_docs/z-wave/device-specific.markdown

```diff
@@ -52,6 +52,25 @@ echo -e -n "...turn on/off string from examples above..." | cu -l /dev/zstick -s
 
 You need to disable the on-board Bluetooth since the board requires the use of the hardware UART (and there's only one on the Pi3). You do this by adding the following to the end of `/boot/config.txt`:
 
+For both processes below you will need to insert your SD card into your PC and open the `/boot/config.txt` file with your favorite text editor.
+
+----------
+The below information is for the Pi 4 specifically
+----------
```

Comment:
Please do not use `--------` to emphasize. Instead, use actual headings or incorporate them into the text normally (which is preferred is the amount of text is low, which is the case here).